app/models/order.rb

```diff
@@ -1,4 +1,10 @@
 class Order < ApplicationRecord
+  module Type
+    SHIPMENT = 'Shipment'.freeze
+    CARRY_OUT = 'CarryOut'.freeze
+    MAX_INTERNATIONAL_ORDER_CODE = 99_999
```

Comment:
I dont think this constant is needed at all
There is no need to maintain a limit to order code
This will be limiting the number of shipment/carryout orders that can be created in the long run